src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java

```diff
@@ -122,32 +122,32 @@ private void testActions() {
 
         ______TS("Typical case: no picture");
 
-        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "Usual Nationality",
+        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "Singaporean",
                                          "male", "this is enough!$%&*</>");
-        profilePage.ensureProfileContains("short.name", "e@email.tmt", "inst", "Usual Nationality",
+        profilePage.ensureProfileContains("short.name", "e@email.tmt", "inst", "Singaporean",
                                           "male", "this is enough!$%&*</>");
         profilePage.verifyStatus(Const.StatusMessages.STUDENT_PROFILE_EDITED);
-        
+
```

Comment:
One possibility for handling legacy data: If the existing value does not  match any items in the drop-down list, choose `--- Select ---` in the dropdown and show the actual value as text beside the dropdown e.g. `current value: abc` .